lib/braille.rb

```diff
@@ -3,16 +3,14 @@ class Braille
               :message
   def initialize(message)
     @dictionary = {}
-    set_braille_dictionary
     @message = message
-    set_grid
```

Comment:
I moved this method out of initialize and added the method directly to the object